src/components/Slot/index.tsx

```diff
@@ -1,3 +1,5 @@
+'use client';
```

Comment:
Isn't this a Next-specific thing?

Does it change anything when using React standalone?